src/array/data_chunk.rs

```diff
@@ -119,7 +119,15 @@ impl fmt::Display for DataChunk {
         use comfy_table::Table;
         let mut table = Table::new();
         if let Some(header) = &self.header {
-            table.set_header(header);
+            match header[0].as_str() {
+                "$insert.row_counts" => {
+                    return write!(f, "{} rows inserted", self.arrays[0].get_to_string(0))
+                }
+                "$create" => return write!(f, "created"),
+                _ => {
+                    table.set_header(header);
+                }
+            }
```

Comment:
Yes I also find it a little weird to put this code in `Display`. What about moving it to main.rs?

https://github.com/risinglightdb/risinglight/blob/5b4af83bcd50592d5675351bd093c12908c31f58/src/main.rs#L58

Reply:
I put it here since there are multiple entry points (`interactive`, `run_sql`, `run_sqllogictest`). And putting here seems ok semantically (`Display` for displaying pretty message). I think if we leave `Display` unchanged, will the dummy pretty tables for `create`, `insert` be otherwise meaningful? 

But I don't have a strong opinion. Both are fine for me.

Reply:
Got your idea. But I think these messages are CLI-specific, and `Display for DataChunk` should show the original structure of a chunk. Currently we can not distinguish between the following 2 chunks 🙈:
```
+---------+  +---------+
| $create |  | created |
+---------+  +---------+
| 0       |
+---------+
```
So I still prefer to move these code into CLI. We can put them in a function if used in multiple places.